StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/GenericInfoScreen/GenericInfoFooterView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  GenericInfoFooterView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 7/2/24.
+//
+
+import Foundation
+import UIKit
+
+func GenericInfoFooterView(
```

Comment:
I extracted this code out of `GenericInfoViewController` because I wanted to have SwiftUI previews for it

That's primarily the reason its in its own file

(the code might be a little funky to look at: it's not a `UIView` subclass, and primarily the reason for that is that because its unnecessary: its just "constructing" a view; although simplicity is the core reason, this can have various side-benefits like: it avoids unnecessary UIView's [performance] and also avoids potential autolayout quirks where embedding this view in another view causes some extra layout considerations)